cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -337,7 +341,7 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
         })) {
           statsOnlyResultTable()
         }
-        p("The resulting role should have privileges that only allow showing roles:")
+        p("The resulting role should have privileges that only allow showing roles, _not privileges_:")
```

Comment:
Maybe give an example?

```suggestion
        p("The resulting role should have privileges that only allows showing roles (e.g. `SHOW ROLES`) , _not privileges_:")
```